### DISCUSSION TITLE
Change PID gains

### DISCUSSION THREAD

**Discussion by CatInTheRain** [larics/icuas22_competition#52] [Q&A] [answered]
Dear organizers,
is it possible in your simulation to change PID gains of the controller between different phases of the competition?

**Comment by fpetric** [comment, marked as answer]
@lmark1 may correct me, but I don't think that is possible. It will not be allowed or possible on the hardware platform, so I suggest to try doing the challenge without such change.

**Reply by CatInTheRain** [reply]
Thanks a lot! In the meantime we have solved our problem so PID gains will be kept constant
